sdk/keyvault/keyvault-keys/test/CRUD.test.ts

```diff
@@ -256,21 +261,23 @@ describe("Keys client - create, read, update and delete operations", () => {
     await testClient.purgeKey(keyName);
   });
 
-  if (isNode && !isPlaybackMode()) {
-    // On playback mode, the tests happen too fast for the timeout to work
-    it("can delete a key with requestOptions timeout", async function() {
-      const keyName = testClient.formatName(`${keyPrefix}-${this!.test!.title}-${keySuffix}`);
-      await client.createKey(keyName, "RSA");
-      await assertThrowsAbortError(async () => {
-        await client.beginDeleteKey(keyName, {
-          ...testPollerProperties,
-          requestOptions: {
-            timeout: 1
-          }
-        });
+  // On playback mode, the tests happen too fast for the timeout to work
+  it("can delete a key with requestOptions timeout", async function() {
+    if (isPlaybackMode() || !isNode) {
+      recorder.skip("Playback and browser test don't timeouts.");
```

Comment:
This won't work, do the following
`recorder.skip(undefined, "Playback for node, browser test(both live and playback) have anomalous behaviour with timeouts.");`.
`recorder.skip` is meant to handle only the record and playback modes.

@sadasant   Also 'To do' - // Add a comment above referring to the logged issue for more info.

Once you change it to the above, `if (isPlaybackMode() || !isNode) {` is not needed.

After the above changes, the test will be skipped in 
- node - record and playback
- browser - record and playback
node-live and browser-live will still be executed.

Ideally, we are not supposed to disable the browser live test.
If the test doesn't work, the CI must fail. That would give us a reason to fix the underlying issue.

In case you want to disable the browser live test, leverage `this.skip()` by checking for the browser.